### DISCUSSION TITLE
Slingbox_server.exe as a service

### DISCUSSION THREAD

**Discussion by getfaxing** [GerryDazoo/Slinger#507] [Q&A] [answered]
Anyone using this as a service in Windows?   Success?  I have not been able to get it working.

**Comment by GerryDazoo** [comment]
Yes.  Yes.  :-(
@sp22222

**Comment by getfaxing** [comment, marked as answer]
The tool to convert the exe to service is "RunAsService", not sure if that is the problem.  I tried the service logon settings as SYSTEM and a specific user account.

**Reply by sp222222** [reply]
https://nssm.cc/ - this is how I run the service on my win box. simple.

**Comment by getfaxing** [comment]
Great! This works.  Problem was the RunAsService tool.   Just a tip, in nssm to add I/O redirection for output and error, so you can easily view the service output and any error msgs if needed.
